server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/step/NewSizeMeasuresStep.java

```diff
@@ -93,45 +97,60 @@ public void execute() {
       FormulaExecutorComponentVisitor.newBuilder(metricRepository, measureRepository)
         .withVariationSupport(periodsHolder)
         .buildFor(ImmutableList.of(duplicationFormula)))
-      .visit(treeRootHolder.getRoot());
+          .visit(treeRootHolder.getRoot());
   }
 
-  private static class NewDuplicationCounter implements Counter<NewDuplicationCounter> {
+  private static class NewSizeCounter implements Counter<NewSizeCounter> {
+    private final MeasureRepository measureRepository;
     private final DuplicationRepository duplicationRepository;
     private final ScmInfoRepository scmInfoRepository;
     private final IntVariationValue.Array newLines = IntVariationValue.newArray();
-    private final IntVariationValue.Array newBlocks = IntVariationValue.newArray();
+    private final IntVariationValue.Array newDuplicatedLines = IntVariationValue.newArray();
+    private final IntVariationValue.Array newDuplicatedBlocks = IntVariationValue.newArray();
 
-    private NewDuplicationCounter(DuplicationRepository duplicationRepository, ScmInfoRepository scmInfoRepository) {
+    private NewSizeCounter(MeasureRepository measureRepository, MetricRepository metricRepository, DuplicationRepository duplicationRepository,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "metricRepository". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1172)